backend/src/api/cycle.js

```diff
@@ -32,6 +36,37 @@ router.get(
   })
 )
 
+// get all cycles belonging to a workspace (either current, outdated, or both)
+// TODO: when authentication server is integrated, only show the cycles
+// to the workspace's owner. Otherwise, 404.
+router.get(
+  '/workspace/:workspaceName',
+  [directorsOnly],
+  errorWrap(async (req, res) => {
+    const workspaceName = req.params.workspaceName
+    const current = req.body.current
+
+    if (!workspaceName) {
+      res.json({
+        code: 400,
+        message: 'malformed request',
+        success: false
+      })
```

Comment:
could be an inline if like what neeraj's PR did

Reply:
I thought you needed to have an empty `return` statement here after the `res.json` so it doesn't get called twice? 

Also not sure if I agree with inline if - I feel like it makes sense within the `render` function of React, but I think it would make the logic harder to follow in most other places. I'm open to discussion on this point though.

Reply:
one thing I'm wondering now is why we don't just do 
```
return res.json({...})
```
and instead change `code` and `message`, which I kind of hate